components/match2/wikis/hearthstone/match_summary.lua

```diff
@@ -32,39 +38,170 @@ end
 function CustomMatchSummary.createBody(match)
 	local showCountdown = match.timestamp ~= DateExt.defaultTimestamp
 
+	CustomMatchSummary._fixGameOpponents(match.games, match.opponents)
+
+	local isTeamMatch = Array.any(match.opponents, function(opponent)
+		return opponent.type == Opponent.team
+	end)
+
 	return MatchSummaryWidgets.Body{children = WidgetUtil.collect(
 		showCountdown and MatchSummaryWidgets.Row{children = DisplayHelper.MatchCountdownBlock(match)} or nil,
-		CustomMatchSummary._isSolo(match) and Array.map(match.games, CustomMatchSummary._createGame) or nil
+		Array.map(match.games, function (game, gameIndex)
+			game.header = match.extradata['subGroup' .. gameIndex .. 'header']
```

Comment:
subgroup =/= game
soubgroup is intended to group games into submatches

so this line here doesn't really make sense to me as it assumes subgroup is always the same as game index and hence subgroup would be redundant

Reply:
Yap, I will leave the submatch implementation for the future, no need to overcomplicate this righ now